src/global/types.h

```diff
@@ -1226,11 +1226,17 @@ typedef struct RGBA8888 {
     uint8_t a;
 } RGBA8888;
 
-typedef struct VECTOR_3D {
+typedef struct XYZ16 {
+    int16_t x;
+    int16_t y;
+    int16_t z;
+} XYZ16;
```

Comment:
`XYZ_16` seems more readable IMO. Or maybe `Vector_16`? Same for 32.

Reply:
Is it OK for me then to convert RGB888 and the family to RGB_888? @walkawayy

Reply:
That makes sense to me. If you agree.